src/nupic/regions/SPRegion.capnp

```diff
@@ -0,0 +1,14 @@
+@0xf4a1336f535a66ba;
+
+using import "/nupic/proto/SpatialPoolerProto.capnp".SpatialPoolerProto;
+
+# Next ID: 7
+struct SPRegionProto {
+  spatialPooler @0 :SpatialPoolerProto;
+  columnCount @1 :UInt32;
+  inputWidth @2 :UInt32;
+  learningMode @3 :UInt32;
```

Comment:
Yes, but the datatype in the region currently is `UInt32`, so I thought I should be consistent.